norddrop/src/device/mod.rs

```diff
@@ -371,6 +373,32 @@ impl NordDropFFI {
         Ok(xfid)
     }
 
+    pub(super) fn set_peers_state(&mut self, states: &str) -> Result<()> {
+        trace!(self.logger, "norddrop_set_peers_state() {:?}", states);
+
+        let states: PeerStates = match serde_json::from_str(states) {
+            Ok(states) => states,
+            Err(e) => {
+                error!(
+                    self.logger,
+                    "Failed to parse set_peer_states() states: {}", e
+                );
+                return Err(ffi::types::NORDDROP_RES_JSON_PARSE);
+            }
+        };
+
+        debug!(self.logger, "Parsed set_peer_states(): {:?}", states);
+
+        let mut instance = self.instance.blocking_lock();
+        let instance = instance
+            .as_mut()
+            .ok_or(ffi::types::NORDDROP_RES_NOT_STARTED)?;
+
+        self.rt.block_on(instance.set_peers_state(states));
```

Comment:
I'm not sure if this is in line with other methods. I think we use `spawn()` not `block_on()` but I don't have a strong opinion on which one we should use. Just saying it's inconsistent

Reply:
I've checked and we actually use `block_on` in other methods: `transfers_since`, `remove_transfer_file`, `new_transfer`, `start`, `stop`, `purge_transfers`, `purge_transfers_until`, so actually all of them :D

Reply:
Oh, then it's all good :+1: . Sorry for not checking it myself

Reply:
no worries at all